README.md

```diff
@@ -206,7 +206,7 @@ See realtime output from `snapctl task watch <task_id>` (CTRL+C to exit)
 
 Stop task:
 ```
-$ $SNAP_PATH/bin/snapctl task stop 02dd7ff4-8106-47e9-8b86-70067cd0a850
+$ $SNAP_PATH/bin/snaptel task stop 02dd7ff4-8106-47e9-8b86-70067cd0a850
```

Comment:
Could you remove $SNAP_PATH in this line?  I'll be good to replace this line with following:
```
$ snaptel task stop 02dd7ff4-8106-47e9-8b86-70067cd0a850
```